src/odemis/odemisd/main.py

```diff
@@ -50,6 +58,36 @@
                     BACKEND_STARTING: 3,
                     }
 
+
+class CustomSafeRepresenter(SafeRepresenter):
+
+    def represent_numpy_float(self, data):
+        data = float(data)
+        self.represent_float(data)
```

Comment:
can you add a comment to explain that this is because yaml does not support dumping numpy floats